scheduler/scheduler.go

```diff
@@ -10,7 +12,7 @@ import (
 type Scheduler struct {
 	logger   lager.Logger
 	config   config.Scheduler
-	backends backend.Backends
+	Backends backend.Backends
```

Comment:
Don't like making this public. That object is used for behaviour and should be independant of the data structure we return in the api.
How about having the Broker return the `config.Backend` instead of reaching in to the schedular? (May require restructuring of the config passed in).